doc/user/layouts/partials/create-source/connector/kafka/with-options.html

```diff
@@ -13,10 +13,10 @@
 
 Field | Value | Description
 ------|-------|------------
```

Comment:
Should we add a column that details required vs optional parameters? Or put optional parameters into their own table?

Reply:
Adding a `Required` column with `t`/`f` and doing the minor rewrite to acknowledge that column would be good! Not necessary to do rn bc what's there's is still accurate

Reply:
I've got to jet off on my second leg of vacation, so merging as is. But agreed this would be nice!